massdash/preprocess/ConformerPreprocessor.py

```diff
@@ -190,18 +184,11 @@ def preprocess(self, window_size: int=175) -> np.ndarray:
         new_data[19] = tmp_arr
 
         ## Add charge state
-        new_data[20] = self.transition_group.targeted_transition_list.PrecursorCharge.values[0] * np.ones(len(data[0]))
+        new_data[20] = self.transition_group.precursor_charge * np.ones(len(data[0]))
         
         ## Convert to float32
         new_data = new_data.astype(np.float32)
         
-        ## trim data if does not match window size starting at the centre
-        if len(new_data[0]) > window_size:
-            middle_index = len(data[0]) // 2
-            trim_start = middle_index - (window_size // 2)
-            trim_end = middle_index + (window_size // 2) + 1
-            new_data = new_data[:, trim_start:trim_end]
```

Comment:
Removing this doesn't break anything? Is it now done in get_peak_boundaries?

Reply:
this should be done in the "pad" method. Possibly pad is not the most appropriate name because it either pads or slices depending on the window size